### ISSUE TITLE
Dosbox-pure doesn't start

### ISSUE THREAD

**Issue by UserGit-2021** [kivutar/rombundler#27]
How to run from 7-zip, zip?
Retroarch supports running from an archive, with a dosbox-pure core.

**Comment by kivutar** [comment]
dosbox-pure itself supports many archive formats, I was able to run QBasic in dosbox-pure in ROMBundler, but I can't remember if I did it in a zip or if I uncompressed it.

**Comment by UserGit-2021** [comment]
> 
> 
> dosbox-pure itself supports many archive formats, I was able to run QBasic in dosbox-pure in ROMBundler, but I can't remember if I did it in a zip or if I uncompressed it.

Will there be support for the flycast core?
Will functions from retroarch be available such as rewind, graphics adjustment (anisotropic filtering, anti-aliasing, etc.), key remapping, quick save?

**Comment by kivutar** [comment]
Yes I will add support for flycast if there are any dreamcast homebrews.

Some features from RetroArch will be added, but not all, only those that make sense for the goal of ROMBundler: `allowing homebrew devs to release their games on Steam or Itch.io as executable.`

**Comment by UserGit-2021** [comment]
> 
> 
> dosbox-pure itself supports many archive formats, I was able to run QBasic in dosbox-pure in ROMBundler, but I can't remember if I did it in a zip or if I uncompressed it.

![изображение](https://user-images.githubusercontent.com/91757058/141965227-a5ca6db1-5c8a-4840-b765-26f01af5ca30.png)
Doesn't start.(

**Comment by kivutar** [comment]
I don't see the extension of your MKTRIL file here.
Also, try to unzip it, just in case.

**Comment by UserGit-2021** [comment]
> 
> 
> I don't see the extension of your MKTRIL file here. Also, try to unzip it, just in case.

![изображение](https://user-images.githubusercontent.com/91757058/141972925-4b4b5bf2-9b0c-4781-b6c3-d76510421ce1.png)
Folder MKTRIL
![изображение](https://user-images.githubusercontent.com/91757058/141972961-3859cc49-44f8-4bda-909e-27e5c0c262c0.png)

**Comment by kivutar** [comment]
OK this seems fine...
Do you know if this program is working in the standalone version of dosbox-pure?
Or in dosbox-pure on retroarch?

**Comment by UserGit-2021** [comment]
>  Or in dosbox-pure on retroarch?
Dosbox-pure on retroarch works with zip. Mortal Kombat Trilogy works via retroarch dosbox-pure.

**Comment by kivutar** [comment]
<img width="1512" alt="Capture d’écran 2022-06-13 à 20 16 56" src="https://user-images.githubusercontent.com/442722/173418483-7b23a731-f2cb-452d-90f1-1a128850b8c6.png">

I'm checking that dosbox_pure is still working, and this is what I got on my M1 macbook
